cmake/README.md

```diff
@@ -0,0 +1,53 @@
+# ESP32CommandStation Cmake project
```

Comment:
This should be renamed to README-cmake.md and moved to top level directory. I'll take care of migrating it to the gh-pages branch for the website after merge.

Reply:
sounds good...